examples/core/transform-feedback-instanced/app.js

```diff
@@ -0,0 +1,348 @@
+/* global window,*/
+/* eslint-disable no-console */
+
+/* eslint-disable max-len */
+// NOTE: This is a port of standard WebGL2 Example : https://github.com/WebGLSamples/WebGL2Samples/blob/master/samples/transform_feedback_instanced.html
+// This example demonstrates how to use luma.gl TransformFeedback API.
+/* eslint-disable max-len */
```

Comment:
Is `eslint-disable max-len` needed? it is listed twice. We already have extended eslint limit.
Just lightly reformat some long shader lines?

Reply:
Needed for the url, it is exceeding limit of 100.